cmd/k8s-device-plugin/main.go

```diff
@@ -129,14 +130,39 @@ func (p *Plugin) ListAndWatch(e *pluginapi.Empty, s pluginapi.DevicePlugin_ListA
 
 	devs := make([]*pluginapi.Device, len(p.AMDGPUs))
 
+	var hw hwloc.Hwloc
+	hw.Init()
+
 	i := 0
 	for id := range p.AMDGPUs {
 		devs[i] = &pluginapi.Device{
 			ID:     id,
 			Health: pluginapi.Healthy,
 		}
+
+		numas, err := hw.GetNUMANodes(id)
+		glog.Infof("Watching GPU with bus ID: %s NUMA Node: %+v", id, numas)
+
+		if err == nil && len(numas) > 0 {
+			numaNodes := make([]*pluginapi.NUMANode, len(numas))
+
+			for j, v := range numas {
+				numaNodes[j] = &pluginapi.NUMANode{
+					ID: int64(v),
+				}
+			}
+
+			devs[i].Topology = &pluginapi.TopologyInfo{
+				Nodes: numaNodes,
+			}
+		} else {
+			glog.Error(err)
```

Comment:
Can hw.getNUMANodes return err == nil and numas == nil? If so what does it mean to glog.Error(nil)?